source/algorithms.tex

```diff
@@ -1650,20 +1652,20 @@
     template<class I, class O>
     using unique_copy_result = copy_result<I, O>;
 
-    template<InputIterator I, Sentinel<I> S, WeaklyIncrementable O,
-             class Proj = identity, IndirectRelation<projected<I, Proj>> C = ranges::equal_to>
-      requires IndirectlyCopyable<I, O> &&
-               (ForwardIterator<I> ||
-                (InputIterator<O> && Same<iter_value_t<I>, iter_value_t<O>>) ||
-                IndirectlyCopyableStorable<I, O>)
+    template<input_iterator I, sentinel_for<I> S, weakly_incrementable O,
+             class Proj = identity, indirect_relation<projected<I, Proj>> C = ranges::equal_to>
+      requires indirectly_copyable<I, O> &&
+               (forward_iterator<I> ||
+                (input_iterator<O> && same_to<iter_value_t<I>, iter_value_t<O>>) ||
```

Comment:
```suggestion
                (input_iterator<O> && same_as<iter_value_t<I>, iter_value_t<O>>) ||
```

Reply:
and on 1667.

Reply:
I've done a global change for this, thanks.